src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,48 @@
+package duke.command;
+
+import java.util.ArrayList;
+
+import duke.exception.MissingArgumentException;
+import duke.storage.TaskList;
+import duke.task.Task;
+
+/**
+ * Command to search task with specific keyword.
+ */
+public class FindCommand extends Command {
+
+    private String request;
+
+    /**
+     * Constructor method to create find command.
+     * @param request User's request to find duke.task.
+     */
+    public FindCommand(String request) {
+        this.request = request;
+    }
+
+    @Override
+    public String execute(TaskList tasks) {
```

Comment:
Do you reckon adding javadocs to all your execute methods would allow the user to make sense of what that method does for that class?  This can be applied to other execute methods in other classes :)

Reply:
I agree i think doing so makes it more explicit and more clear for developers to understand what the execute does for each of the commands